integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
@chenhanxiao I was thinking about something like this:
```
    ...
    _, out, err := buildImageWithOut(name,
        `FROM scratch
        EXPOSE 192.168.1.2:2375:5678`,
        true)
    if err != nil {
        t.Fatal(err)
    }

    if !strings.Contains(out, "Dockerfile EXPOSE (ip:hostPort:containerPort) has been deprecated") {
        t.Fatal("Missing warning message")
    }
    ...
```
this seems to PASS for me.  Does that make sense?